### ISSUE TITLE
Question about use MultipleNegativesRankingLoss for customed sts dataset

### ISSUE THREAD

**Issue by ewayuan** [UKPLab/sentence-transformers#1307]
Hi, I'm trying to use MultipleNegativesRankingLoss as a loss function for my customed dataset, which contains (s1, s2, similarity-score), (s1, s2, similarity-score), ..., (s_i, s_j, similarity-score), where similarity-score is calculated by cross-encoder. And I found the following example: https://github.com/UKPLab/sentence-transformers/blob/42c1d69a96a75ce6594da6335b535cac99ee91d3/examples/training/paraphrases/training.py

and also one script example using cosine-similarity loss: https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/sts/training_stsbenchmark_continue_training.py

Is that I can simply chang cosine-similarity loss to MultipleNegativesRankingLoss? Or do I need to use MultiDatasetDataLoader to specify the number of epochs? What's the purpose of MultiDatasetDataLoader? How could I fill the batch_size_pairs in MultiDatasetDataLoader?
